.spelling

```diff
@@ -304,6 +305,7 @@ IPs
 iptables
 IPv4
 IPv6
+istio
```

Comment:
Remove this, it should not be needed.

Reply:
was needed in local runs.

Reply:
Hmm, where is it used? Can you instead change the place that uses it to uppercase Istio, or `istio` if that's more appropriate? We have tried very hard to make sure all uses of Istio in the docs are uppercase.